branchtest/branchtest/Source.cpp

```diff
@@ -1,6 +1,19 @@
 #include <iostream>
-#define num 10000
+#define num 1000
+int func();
```

Comment:
**dis number bigger**
@jonjonjovi 
[https://www.youtube.com/](url)
- [make it bigger ] 
`#define num 10000`

Reply:
[https://www.learncpp.com/](url)